python/servo/bootstrap.py

```diff
@@ -21,33 +21,6 @@
 from servo.util import extract, download_file, host_triple
 
 
-def install_trusty_deps(force):
```

Comment:
The Ubuntu 14.04 section of the readme was removed recently because it has been EOL for non-paid users for a year now.